### DISCUSSION TITLE
Which method in the plug-in can get the less file and return the modified file。🙏🙏🙏

### DISCUSSION THREAD

**Discussion by xiaoxiaohuayu** [web-infra-dev/rsbuild#3869] [Q&A]
There is a history project in which less has a syntax error.，I have a problem here. I need to write a plug-in to get less files and convert iconfont-size-under-12px (number px) into. iconfont-size-under-12px(unit (number, px)), but I don't know which life cycle to use to get less files to match and replace

``` × Module build failed:
  ╰─▶   ×
        │
        │     .@{tag-close-prefix-cls} {
        │         .iconfont-size-under-12px(20px);
        │       ^
        │ Error evaluating function `unit`: the first argument to unit must be a number. Have you forgotten parenthesis?
        │       Error in /Users/hao/Documents/ETC/param-web/node_modules/view-design/src/styles/components/tag.less (line 187, column 8)
        │```

**Comment by xiaoxiaohuayu** [comment]
有大佬能回答一下吗？

**Comment by 9aoy** [comment]
try https://rsbuild.dev/plugins/dev/core#apitransform

**Reply by xiaoxiaohuayu** [reply]
试过了不行。的确是能扫到 `/node_modules/view-design/src/styles/index.less`。但是报错的文件是在组件UI库中的index.less里面引用的`@import "./components/index";`中的其中一个文件报错的。 api.transform扫不到`node_modules/view-design/src/styles/components/tag.less`文件。还有其他的方法吗？我想的是在编译之前就能拿到这些less文件，然后再去修改。或者大佬有更好的、更快的处理方式。rsbuild的方法实在是太丰富。还需要慢慢探索。感谢🙏你们为开源作出的贡献
